docs/api/using-api.asciidoc

```diff
@@ -34,13 +34,39 @@ For example, the following `curl` command exports a dashboard:
 curl -X POST -u $USER:$PASSWORD "localhost:5601/api/kibana/dashboards/export?dashboard=942dcef0-b2cd-11e8-ad8e-85441f0c2e5c"
 --
 
-The following {kib} APIs are available:
+[float]
+[[api-request-headers]]
+=== Request headers
+
+For all non whitelisted APIs, you must use a header.
```

Comment:
Whitelisted APIs will still require the `Content-Type: application/json` request header.
Something I forgot to mention in my earlier comment: `Content-Type` is only applicable when sending a payload as part of the API request. These are _typically_ the same endpoints which require `kbn-xsrf`